hasura.planx.uk/migrations/1692609979701_create_table_public_user_role_enum/up.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE "public"."user_roles" (
+  "value" text NOT NULL,
+  PRIMARY KEY ("value"),
+  UNIQUE ("value")
+);
+
+COMMENT ON TABLE "public"."user_roles" IS E'Enum of possible user roles in PlanX';
+
+INSERT INTO
+  "public"."user_roles"("value")
+VALUES
+  (E'admin'),
+  (E'viewer');
```

Comment:
I'm a little confused - is `admin` here the same as Hasura's reserved `admin`? 

My original understanding was that Hasura reserved roles `admin` & `public` won't actually be included in this user_roles enum, but things like `Flow Editor`, `Flow Viewer` would be.

Can we perhaps align on an MVP of role names before this is merged and try to update originally proposed table accordingly? https://docs.google.com/document/d/1Vui62_7iHuT60-FrMJATrhqsj2nQZ71N5fEX1nfZnvU/edit

Reply:
Apologies - this is me being a bit clumsy with naming things. I've updated this to `teamAdmin` and `teamViewer`.

The intention was to later map admin -> teamAdmin, and viewer -> teamViewer when building the JWT but we can skip that step by just using consistent naming. The reasoning for this was that the `team` prefix was a bit superfluous in the context of this table but looking back it's just unclear / confusing.

If we end up with flow-level permissions in addition to team-level permissions, we would then be able to add a `flow_id` column here (or make another table for this?) and then have `flowViewer` and `flowAdmin` enums added.